pkg/cmd/trades.go

```diff
@@ -101,15 +101,17 @@ var tradesCmd = &cobra.Command{
 
 		log.Infof("%d trades", len(trades))
 		for _, trade := range trades {
-			log.Infof("TRADE %s %s %4s %s @ %s orderID %d %s amount %f",
+			log.Infof("TRADE %s %s %4s %s @ %s orderID %d %s amount %f , fee %s %f ",
 				trade.Exchange.String(),
 				trade.Symbol,
 				trade.Side,
 				util.FormatFloat(trade.Quantity, 4),
 				util.FormatFloat(trade.Price, 3),
 				trade.OrderID,
 				trade.Time.Time().Format(time.StampMilli),
-				trade.QuoteQuantity)
+				trade.QuoteQuantity,
+				trade.FeeCurrency,
+				trade.Fee)
```

Comment:
use trade.Fee then trade.FeeCurrency, so it show display something like 0.001 BNB ? looks better?

Reply:
ok